### DISCUSSION TITLE
请问这个可以用来做邮箱验证吗？

### DISCUSSION THREAD

**Discussion by iomeone** [Jinnrry/PMail#79] [General]
比如有个api ，可以调用然后向用户的邮箱发送 密码重置链接， 请问这个可以做到吗？

**Comment by Jinnrry** [comment]
https://github.com/Jinnrry/PMail/wiki/

有发送邮件api，但是这个项目没有做频率管理，你需要自己控制发送频率，不然大量发送很容易被其他邮箱拉黑
